src/lib/Components/States/ZeroState.tsx

```diff
@@ -18,7 +18,7 @@ export const ZeroState = (props: ZeroStateProps) => (
 
     {!!props.subtitle && (
       <>
-        <Sans size="4" textAlign="center">
+        <Sans size="4" textAlign="center" maxWidth={!!props.title ? "100%" : "80%"}>
```

Comment:
nitpick, nonblocking:
```suggestion
        <Sans size="4" textAlign="center" maxWidth={props.title ? "100%" : "80%"}>
```